README.md

```diff
@@ -22,11 +22,11 @@ The dataset is based on [Version 9](https://zenodo.org/records/10118572) of [`Eu
 
 For `EuroCropsML`, we acquired and aggregated data for the following countries:
 
-| Country      | Total number of datapoints for Sentinel-2 | Total number of datapoints for Sentinel-1 | Number of distinct classes | 
-|--------------|-------------------------------------------|-------------------------------------------|----------------------------|
-| Estonia      | 175,906                                   | 154,742                                    | 127                        |
-| Latvia       | 431,143                                   |                                           | 103                        |
-| Portugal     | 99,634                                    |                                           | 79                         |
+| Country      | Number of distinct classes | Total number of datapoints for Sentinel-2 | Total number of datapoints for Sentinel-1 | 
+|--------------|----------------------------|-------------------------------------------|-------------------------------------------|
+| Estonia      | 127                        | 175,906                                   | 176,055                                   | 
+| Latvia       | 103                        | 431,143                                   | -                                         | 
+| Portugal     | 79                         | 99,634                                    | -                                         | 
```

Comment:
The total number **706,683** in line 19 above still refers to Sentinel-2 after cloud filtering, correct? Do we want to keep it like that?

Reply:
Yes, it refers to the final number of Sentinel-2 parcels. I added the number of S1 data points.